java/java.completion/src/org/netbeans/modules/java/completion/JavaCompletionTask.java

```diff
@@ -3930,6 +4136,8 @@ public boolean accept(Element e, TypeMirror t) {
             }
         };
         for (TypeElement e : controller.getElementUtilities().getGlobalTypes(acceptor)) {
+            if (e.getQualifiedName().toString().equals(JAVA_LANG_RECORD) && env.getController().getSourceVersion().compareTo(SOURCE_VERSION_RELEASE_14) < 0) 
```

Comment:
If I don't add the check , 'Record' entry need to be added in old golden files (roughly 65 files) otherwise existing test-cases will fail with nb-javac for 14

Reply:
Sorry, I am unclear which files - the code completion tests run and pass on JDK 14 without nb-javac, the golden files have been updated some time ago. I believe files like ".../JavaCompletionTaskTest/14/initBlockTypesAndLocalMembers.pass" have been specifically added so that the tests would pass on JDK 14. Not sure if nb-javac should change anything here?

Reply:
java.lang.Record check is now removed. The problem was the in the proposed nb-javac-14 where it was added by mistake,corrected now